### PR TITLE
New features added

### PR DESCRIPTION
**Pull request by AnkitSharma-007**
Added the following two new features:

- Delete button
- Tag module

### CODE REVIEW

File: projects/ngx-bulma/tag/tag.component.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    '[class.is-primary]': 'theme === "primary"',
+    '[class.is-link]': 'theme === "link"',
+    '[class.is-info]': 'theme === "info"',
+    '[class.is-success]': 'theme === "success"',
+    '[class.is-warning]': 'theme === "warning"',
+    '[class.is-danger]': 'theme === "danger"',
+    '[class.is-normal]': 'size === "normal"',
+    '[class.is-medium]': 'size === "medium"',
+    '[class.is-large]': 'size === "large"',
+    '[class.is-rounded]': 'rounded === "true"',
```
Comment (santoshyadavdev) [inline projects/ngx-bulma/tag/tag.component.ts:31]: rounded === true

File: projects/ngx-bulma/tag/tag.component.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    '[class.is-delete]': 'delete === "true"',
```
Comment (santoshyadavdev) [inline projects/ngx-bulma/tag/tag.component.ts:32]: delete === true

File: projects/ngx-bulma/tag/tag.component.ts (hunk 1 above)
Comment (AnkitSharma-007) [reply projects/ngx-bulma/tag/tag.component.ts:31]: ``` '[class.is-rounded]': 'rounded === "true"',``` doesnot work. Therefore, I make is as ``` '[class.is-rounded]': 'rounded'```

File: projects/ngx-bulma/tag/tag.component.ts (hunk 2 above)
Comment (AnkitSharma-007) [reply projects/ngx-bulma/tag/tag.component.ts]: similar change as previous comment.

**Pull request merged**